lib/theme_check/language_server/handler.rb

```diff
@@ -99,7 +99,7 @@ def text_document_uri(params)
       end
 
       def path_from_uri(uri)
-        uri&.sub('file://', '')
+        uri&.sub('file://', '')&.sub('/c%3A', '')
```

Comment:
don't you really want `.chomp`? Or are you expecting \r in the middle of the string?